python/modules/testgenjet.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python                                                    
```

Comment:
What is the purpose of this script? If this is a user script, can you keep it in your own repo, so that we can keep the central code clean. Thanks